rental.rb

```diff
@@ -0,0 +1,15 @@
+# Rental class
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] The `date` variable here should have getter and setter (`attr_accessor`), not just getter (`attr_reader`). Kindly refactor accordingly.

Reply:
The `date` variable already has `attr_accessor` though.

Reply:
@Yothu sorry about that. Thanks for pointing it out. It was intended the other way. `:person` and `:book` should have getter and setter instead, not just getter

Reply:
This issue is well resolved! ✔️